hathor/builder/cli_builder.py

```diff
@@ -322,6 +320,7 @@ def create_manager(self, reactor: Reactor) -> HathorManager:
             verification_service=verification_service,
             cpu_mining_service=cpu_mining_service,
             execution_manager=execution_manager,
+            peer_id_json_path=self._args.peer,
```

Comment:
Because the created `PeerId` is used by other classes that receive it during building, so it must be ready before `HathorManager`. However, I changed this so the file path is persisted in the `PeerId` itself, simplifying things and allowing me to remove this argument from the manager.

Reply:
Got it. Thanks!